### PR TITLE
@mzikherman: Adds eslint; fix lint errors

### PR DESCRIPTION
**Pull request by dzucconi**

### CODE REVIEW

File: schema/partner.js [hunk 1]
```diff
@@ -9,69 +9,76 @@ import {
...
     },
     profile: {
       type: Profile.type,
-      resolve: ({ default_profile_id }) => gravity(`profile/${default_profile_id}`)
+      resolve: ({ default_profile_id }) => gravity(`profile/${default_profile_id}`),
     },
     type: {
-      type: GraphQLString
+      type: GraphQLString,
+      deprecationReason: 'Avoid naming fields `type`',
```
Comment (dzucconi) [inline schema/partner.js]: This results in a parse error currently:



I'm just deprecating it and then I'll change the caller in the show2 routes. Then we can remove it and run eslint as a buildstep

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "env": {
+    "browser": true,
+    "node": true,
+    "mocha": true,
+    "es6": true
+  },
+  "extends": "airbnb",
+  "rules": {
+    "camelcase": 0
```
Comment (mzikherman) [inline .eslintrc.json]: :snake:

File: config.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
 export default {
   REQUEST_THROTTLE_MS: 5000,
-  REQUEST_TIMEOUT_MS: 5000
+  REQUEST_TIMEOUT_MS: 5000,
```
Comment (mzikherman) [inline config.js]: ? isn't this the last item in the object

File: schema/partner.js (hunk 1 above)
Comment (dzucconi) [inline schema/partner.js]: Oh I'm an idiot and missed the actual `type` declaration above. I can remove this + kind.

File: lib/apis/fetch.js [hunk 4]
```diff
@@ -4,14 +4,16 @@ import config from '../../config';
 
 export default (url, options = {}) => {
   return new Promise((resolve, reject) => {
-    request(url, _.defaults(options, { method: 'GET', timeout: config.REQUEST_TIMEOUT_MS }), (err, response) => {
+    const opts = _.defaults(options, { method: 'GET', timeout: config.REQUEST_TIMEOUT_MS });
```
Comment (mzikherman) [inline lib/apis/fetch.js]: nice

File: config.js (hunk 3 above)
Comment (dzucconi) [inline config.js]: Welcome to the world of comma dangle.

It's sorta ugly but it's pretty useful when 90% of the code consists of objects.

Comment (dzucconi) [inline config.js]: See: https://github.com/airbnb/javascript#19.2

File: schema/fair.js [hunk 5]
```diff
@@ -6,67 +6,72 @@ import {
...
     start_at: date,
     end_at: date,
     name: {
-      type: GraphQLString
+      type: GraphQLString,
     },
     published: {
-      type: GraphQLBoolean
+      type: GraphQLBoolean,
+      deprecationReason: 'Prefix Boolean returning fields with `is_`',
```
Comment (mzikherman) [inline schema/fair.js]: cool

Comment (dzucconi) [inline schema/fair.js]: I mostly wanted to try this out. Pretty cool; you just give it a deprecationReason and it stops showing up in graphiql, but will still work if you query with it.

File: config.js (hunk 3 above)
Comment (mzikherman) [inline config.js]: got it,

**Comment by mzikherman** [comment]
Epic :balloon:

File: schema/fair.js (hunk 5 above)
Comment (craigspaeth) [inline schema/fair.js]: w00t w00t! Was curious what GraphQL's deprecations looked like in implementation.

**Pull request merged**